.travis.yml

```diff
@@ -7,6 +7,7 @@ python:
   - "3.4"
   - "3.5"
   - "3.6"
+  - "3.7"

```

Comment:
Can you set 3.7 to use the ``xenial`` distribution? See here: https://docs.travis-ci.com/user/languages/python/